server/src/core/messages.ts

```diff
@@ -3,9 +3,12 @@ export const ERROR_OPERATION_FAILED = 'Operation failed';
 export const ERROR_LOGIN_FAILED = 'Login failed';
 export const ERROR_INVALID_ACCESS_TOKEN = 'Invalid access token';
 export const ERROR_RESOURCE_NOT_FOUND = 'Resource not found';
+export const ERROR_TRANSACTION_REQUESTED = 'No such transaction was initiated';
+export const ERROR_TRANSACTION_NOT_FOUND = 'Transaction not found';
 export const ERROR_USER_NOT_FOUND = 'User not found';
 export const ERROR_SERVER_ERROR = 'Server error occurred';
 export const ERROR_ROUTE_NOT_FOUND = 'Route does not exist';
 export const ERROR_PHONE_ALREADY_IN_USE = 'The specified phone number is already in use';
+export const ERROR_AT_API_ERROR = 'AfricasTalking error occurred';
 export const ERROR_BENEFICIARY_NOMINATION_FAILED = 'Failed to nominate the specified user as a beneficiary';
-export const ERROR_USER_CANNOT_BE_BENEFICIARY = 'The specified used cannot be nominated as beneficiary';
\ No newline at end of file
+export const ERROR_USER_CANNOT_BE_BENEFICIARY = 'The specified used cannot be nominated as beneficiary';
```

Comment:
The message here should be `The specified user cannot be nominated as beneficiary`, not `The specified used cannot be nominated as benefiary`